File: Projects/ThematicCalculator/app/src/main/java/dam_a45125/thematiccalculator/Calculator.java
package dam_a45125.thematiccalculator;

public class Calculator {


    public double sum(double num1, double num2){
        return num1 + num2;
    }

    public double sub(double num1, double num2){
        return num1 - num2;
    }

    public double mult(double num1, double num2){
        return num1 * num2;
    }

    public double div(double num1, double num2){
        return num1/num2;
    }

    public double cosseno(double num1){return Math.cos(num1);}

    public double seno(double num1){return Math.sin(num1);}

    public double tangente(double num1){ return Math.tan(num1);}

    public double module(double num1, double num2){return num1 % num2;}

}

File: Projects/ThematicCalculator/app/src/main/java/dam_a45125/thematiccalculator/Calculator.kt
package dam_a45125.thematiccalculator

class Calculator {

    fun sum(num1: Double, num2: Double): Double {
        return num1 + num2
    }

    fun sub(num1: Double, num2: Double): Double {
        return num1 - num2
    }

    fun mult(num1: Double, num2: Double): Double {
        return num1 * num2
    }

    fun div(num1: Double, num2: Double): Double {
        return num1 / num2
    }

    fun cosseno(num1: Double): Double {
        return Math.cos(num1)
    }

    fun seno(num1: Double): Double {
        return Math.sin(num1)
    }

    fun tangente(num1: Double): Double {
        return Math.tan(num1)
    }

    fun module(num1: Double, num2: Double): Double {
        return num1 % num2
    }
}